lib/explorer/series.ex

```diff
@@ -6550,14 +6620,23 @@ defmodule Explorer.Series do
     :"#{backend}.Series"
   end
 
-  defp dtype_error(function, dtype, valid_dtypes) when is_list(valid_dtypes) do
+  defp super_dtype_error(function, dtype, valid_super_dtypes) do
     raise(
       ArgumentError,
       "Explorer.Series.#{function} not implemented for dtype #{inspect(dtype)}. " <>
-        "Valid " <> Shared.inspect_dtypes(valid_dtypes, with_prefix: true)
+        "Valid dtypes are any subtype of #{inspect(valid_super_dtypes)}"
```

Comment:
My attempt to handle the fact that we can't list all the dtypes easily anymore. Open to ideas on this.